x-pack/plugins/ml/public/data_frame/components/group_by_list/group_by_label_summary.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+
+import { PivotGroupByConfig } from '../../common';
+
+interface Props {
+  item: PivotGroupByConfig;
+  optionsDataId: string;
+}
+
+export const GroupByLabelSummary: React.SFC<Props> = ({ item, optionsDataId }) => {
+  return 'interval' in item ? (
+    <EuiFlexGroup alignItems="center" gutterSize="s">
+      <EuiFlexItem>{optionsDataId}</EuiFlexItem>
+      <EuiFlexItem grow={false} style={{ color: '#999' }}>
```

Comment:
Are there any of the EUI colours you can use here? I wonder how this would look if using dark theme for instance?

Reply:
Oh yes that definitely needs to get rid of the hard-coded color! I changed this to use `<EuiTextColor color="subdued">{item.interval}</EuiTextColor>` instead. It's a bit darker but supports dark theme and is more in line with EUIs accessibility requirements. I also added unit test snapshot variations to cover the rendering of different aggregation types. Fixed in d8a33e8.